benchmark_runner/common/template_operations/templates/scale/state_signals_exporter_pod.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: state-signals-exporter
+  namespace: {{ namespace }}
+  labels:
+    app: state-signals-exporter
+    type: state-signals-exporter
+    benchmark-uuid: {{ uuid }}
+spec:
+  selector:
+    matchLabels:
+      app: state-signals-exporter
+  {%- if pin_node1 %}
+  nodeSelector:
+    kubernetes.io/hostname: {{ pin_node1 }}
+  {%- endif %}
+  resources:
+    requests:
+      cpu: 10m
+      memory: 10Mi
+    limits:
+      cpu: 2
+      memory: 2Gi
+  containers:
+    - name: state-signals-exporter-pod
+      namespace: {{ namespace }}
+      image: quay.io/ebattat/state-signals-exporter:v0.5.2
+      imagePullPolicy: "IfNotPresent"
+      env:
+        - name: REDIS_HOST
+          value: "{{ redis }}"
+        - name: SCALE_NUM
+          value: "{{ scale }}"
+        - name: TIMEOUT
+          value: "{{ timeout }}"
+      command: ["/bin/bash"]
+      args: ["-c", "python3.9 /state_signals_exporter.py $REDIS_HOST $SCALE_NUM $TIMEOUT"]
```

Comment:
No need to invoke a shell here; just invoke the python command (via /usr/bin/env to do the path search).  That will avoid problems if any of the variables have spaces in them (they shouldn't, but it's best not to take that risk).

```
command: "/usr/bin/env"
args: ["python3", "/state_signals_exporter.py", "$REDIS_HOST", "$SCALE_NUM", "$TIMEOUT"]
```

Reply:
It's safer to run it as I suggested, since it avoids problems with spaces in any of the arguments.

Reply:
It really isn't correct.  If you must do it this way:

```
args: ["-c", "python3.9 /state_signals_exporter.py '$REDIS_HOST' '$SCALE_NUM' '$TIMEOUT'"]
```
would be better, although it's still not completely free of hazards.

Reply:
Fixed